Makefile

```diff
@@ -0,0 +1,24 @@
+SHELL := /usr/bin/env bash
+
+release:
```

Comment:
I copied this from here: https://github.com/cucumber/cucumber-ruby/blob/master/Makefile Should I be using another target?

Reply:
Actually, the `release` target here does not release anything. It runs the docker image with the secrets.

In `cucumber-ruby`, the `release` target actually releases `cucumber-ruby` using the docker image with the secrets.

I would prefer to remain consistent with the monorepo and then having at least two targets: `docker-run-with-secrets` and `release`. 

If it looks like `cucumber-ruby` with a single `release` target which is a all-in-one, it would be ok.